himbaechel/uarch/gatemate/pack_mult.cc

```diff
@@ -916,6 +916,43 @@ void GateMatePacker::pack_mult()
             auto *cpe_half = (p % 2 == 1) ? mult_cell.upper : mult_cell.lower;
 
             mult->movePortTo(ctx->idf("P[%d]", p), cpe_half, id_CPOUT);
+
+            auto *cpe_half_cpout = cpe_half->getPort(id_CPOUT);
+            if (cpe_half_cpout && cpe_half_cpout->users.entries() == 1) {
+                auto cpe_half_cpout_user = *cpe_half_cpout->users.begin();
+                auto *dff = cpe_half_cpout_user.cell;
+                NPNR_ASSERT(dff != nullptr);
+                if (dff->type == id_CC_DFF) {
+                    // Instantiate a P passthrough L2T4 for the flop.
+                    auto *p_passthru =
+                            create_cell_ptr(id_CPE_L2T4, ctx->idf("%s$p[%d]_passthru", cpe_half->name.c_str(ctx), p));
+
+                    p_passthru->params[id_INIT_L00] = Property(LUT_D0, 4);   // IN5
```

Comment:
maybe better get rid of these comments in these 3 lines (for id_INIT_L* params), since those are not correct for all cases